### ISSUE TITLE
Show "X items out of stock" on the Stock Exit page for a depot

### ISSUE THREAD

**Issue by jniles** [third-culture-software/bhima#4381]
We have a large empty space in the stock exit module.  Here it is:
![image](https://user-images.githubusercontent.com/896472/80474541-7b221e80-893f-11ea-86d2-69e8456091e6.png)
_Fig 1: Stock Exit Module empty area_

I propose that we make better use of this area by adding in a list of items that are out of stock ordered by the most recent items that have gone out of stock.  It would look something like this:
![image](https://user-images.githubusercontent.com/896472/80475588-fafcb880-8940-11ea-9f6f-f5ae4f417a5a.png)

Basically, list the 5 most recent stockouts, and then provide a link to the "articles in stock" for that depot filtered on items out of stock.  This will hopefully prompt our users to do something about their stockouts rather than simply sit with incorrect stock counts in their depot.
